source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -0,0 +1,414 @@
+		qDebug() << "Checking start up value from the OLD Horizontal DownStream as " << horizontalDownstreamControl_->value();
+		onHorizontalControlValueChanged(horizontalUpstreamControl_->value());
+		qDebug() << "Checking start up value from the OLD Horizontal upstream feedback as " << horizontalUpstreamFeedbackControl_->value();
+		qDebug() << "Checking start up value from the OLD Horizontal DownStream feedback as " << horizontalDownstreamFeedbackControl_->value();
+		onHorizontalFeedbackControlValueChanged(horizontalUpstreamFeedbackControl_->value());
+		qDebug() << "Checking start up value from the OLD Horizontal upstream status as " << horizontalUpstreamStatusControl_->value();
+		qDebug() << "Checking start up value from the OLD Horizontal DownStream status as " << horizontalDownstreamStatusControl_->value();
+		onHorizontalStatusControlValueChanged(horizontalUpstreamStatusControl_->value());
+
+
+		// Max / Min value?
+	}
+}
+
+void BioXASSideTableCoordinator::onVerticalControlValueChanged(double value)
+{
+	Q_UNUSED(value)
+	if(!connectedOnce_)
+		return;
+
+	double upstreamHeight = verticalUpstreamControl_->value();
+	double downstreamHeight = verticalDownstreamControl_->value();
+	manipulateVerticalPVChange(upstreamHeight, downstreamHeight, softIOCHeightControl_, softIOCPitchControl_);
+}
+
+void BioXASSideTableCoordinator::onVerticalFeedbackControlValueChanged(double value)
+{
+	Q_UNUSED(value)
+	if(!connectedOnce_)
+		return;
+
+	double upstreamHeight = verticalUpstreamFeedbackControl_->value();
+	double downstreamHeight = verticalDownstreamFeedbackControl_->value();
+	manipulateVerticalPVChange(upstreamHeight, downstreamHeight, softIOCHeightFeedbackControl_, softIOCPitchFeedbackControl_);
+}
+
+void BioXASSideTableCoordinator::onVerticalStatusControlValueChanged(double value)
+{
+	Q_UNUSED(value)
+	if(!connectedOnce_)
+		return;
+
+	// Status value: 0 - "MOVE DONE";  1 - "MOVE ACTIVE"; 2 - "AT LIMIT" ("MINOR"); 3 - "FORCED STOP" ("MINOR"); 4 - "ERROR" ("MAJOR")
+	// Status is "MOVE ACTIVE" if at least one motor is moving or the largest status value
+	double status = 0;
+	if (verticalUpstreamStatusControl_->withinTolerance(1) || verticalDownstreamStatusControl_->withinTolerance(1)) {
+		status = 1;
+	} else {
+		double verticalUpstreamControlStatus = verticalUpstreamStatusControl_->value();
+		double verticalDownstreamControlStatus = verticalDownstreamStatusControl_->value();
+		status = verticalUpstreamControlStatus > verticalDownstreamControlStatus ? verticalUpstreamControlStatus : verticalDownstreamControlStatus;
+	}
+
+	if (!softIOCHeightStatusControl_->withinTolerance(status)) {
+		softIOCHeightStatusControl_->move(status);
+	}
+	if (!softIOCPitchStatusControl_->withinTolerance(status)) {
+		softIOCPitchStatusControl_->move(status);
+	}
+}
+
+
+void BioXASSideTableCoordinator::onHorizontalControlValueChanged(double value)
+{
+	Q_UNUSED(value)
```

Comment:
All of these functions don't need the ```double value``` argument for the slot connection to work.  Slots can have less arguments than the signals that connect them.  I'd remove the value since and the Q_UNUSED for code simplicity.